templates/overrides/infrastructure-vsphere/v0.6.4/cluster-template.yaml

```diff
@@ -0,0 +1,207 @@
+---
+apiVersion: cluster.x-k8s.io/v1alpha3
```

Comment:
this isn't needed as it's already published on the release

Reply:
will remove and provide link to the file